.vscode/launch.json

```diff
@@ -0,0 +1,37 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Remote Attach",
+            "type": "python",
+            "request": "attach",
+            "connect": {
+                "host": "0.0.0.0",
+                "port": 5678
+            },
+            "pathMappings": [
+                {
+                    "localRoot": "${workspaceFolder}",
+                    "remoteRoot": "/workspace/axolotl/"
+                }
+            ],
+            "justMyCode": false
+        },
+        {
+            "name": "train",
+            "type": "python",
+            "request": "launch",
+            "module": "accelerate.commands.launch",
+            "args": [
+                "${workspaceFolder}/scripts/finetune.py",
+                // "${file}",
+                "${workspaceFolder}/examples/llama-2/tiny-random.yml",
+            ], // other args comes after train.py
+            "console": "integratedTerminal",
+            // "env": {"CUDA_LAUNCH_BLOCKING": "1"}
+          },
+    ]
+}
```

Comment:
Can we put this into a README rather than hijacking the IDE without notice?

Reply:
What do you mean hijacking the IDE without notice? The point of this config file is to inform VSCode IDE there are available debugging options to use.

That said, I planned to remove the `Python: Remote Attach` launch option because it requires manually enabling `debugpy` in `scripts/finetune.py` (also to be removed). I personally used this for debugging inside a Docker container.

The `train` launch config is useful though for anyone using VSCode and I'd remove line 31 in favour of 30 with dynamic `${file}`.

Reply:
Deleting file for now. We can discuss this in a future PR